climada/hazard/centroids/centr.py

```diff
@@ -140,9 +142,9 @@ def check(self):
                 LOGGER.error('Missing meta information: width, height,'
                              'crs or transform')
                 raise ValueError
-            if self.meta['transform'][4] > 0:
-                LOGGER.error('Meta does not contain upper left corner data.')
-                raise ValueError
+            if self.meta['transform'][4] > 0 or self.meta['transform'][0] < 0:
```

Comment:
What are `self.meta['transform'][4]` and `self.meta['transform'][0]` ? Maybe defining variables for these makes is understandable.

Reply:
Agreed and changed!